pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -732,6 +736,9 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(logger klog.Logger, pInfo *
 	// In this case, we try to requeue this Pod to activeQ/backoffQ.
 	queue := p.requeuePodViaQueueingHint(logger, pInfo, schedulingHint, ScheduleAttemptFailure)
 	logger.V(5).Info("Pod moved to an internal scheduling queue", "pod", klog.KObj(pod), "event", ScheduleAttemptFailure, "queue", queue, "schedulingCycle", podSchedulingCycle)
+	if queue == activeQ {
+		p.cond.Broadcast()
+	}
```

Comment:
TestRequeueByBindFailure doesn't pass without this change.
It's a bug and I have another PR to fix this asap:
https://github.com/kubernetes/kubernetes/pull/119784